split.Console/Models/Blackjack/BlackjackCard.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackCard
+    {
+        public SuitType Suit;
+        public int Rank;        //TODO Decide if this would be better as an enum
+
+        public BlackjackCard(SuitType suit, int rank)
```

Comment:
Also, to be fair, I think cards in all games, not just Blackjack, would probably contain the ```Suit``` and the ```Rank```. Maybe it would be better to just have a ```Card``` class?